js/src/forum/components/ReplyPlaceholder.js

```diff
@@ -25,6 +26,7 @@ export default class ReplyPlaceholder extends Component {
                 {avatar(app.session.user, { className: 'PostUser-avatar' })}
                 {username(app.session.user)}
               </h3>
+              <ul className="PostUser-badges badges">{listItems(app.session.user.badges().toArray())}</ul>
```

Comment:
Another concern: we're reusing CSS from an unrelated component here. This is fairly common in official extensions, but I'd like to keep core's CSS as organized as we can here.

Reply:
As this change is to retain consistency with the rest of the post stream, I'd put forward that re-using the CSS is valid in this scenario, otherwise there is a (small) overhead of maintaining two identical CSS classes?

Reply:
Looking at this again, I see that other parts of this component are already using the `PostUser` css class so my concern is invalid